### PR TITLE
Miscellaneous changes and fixes

### PR DESCRIPTION
**Pull request by Tiagoquix**

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -1338,11 +1339,13 @@ alias sound_ultra "snd_disable_mixer_duck 0;snd_pitchquality 1;dsp_slow_cpu 0;sn
...
-// ------------------------
-// '-- Steam Controller --'
-// ------------------------
-// Controls Steam Controller/Steam Input
+// ----------------------------------
+// '-- Steam Controller/Joysticks --'
+// ----------------------------------
+// Controls Steam Controller/Steam Input and Joysticks
 
+joystick 0 // Disable joystick support
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Just having the launch option is fine.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 1 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: Done.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 2]
```diff
@@ -1343,6 +1344,7 @@ alias sound_ultra "snd_disable_mixer_duck 0;snd_pitchquality 1;dsp_slow_cpu 0;sn
 // ------------------------
 // Controls Steam Controller/Steam Input
 
+sc_disable 1 // Disable Steam Controller API
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Same for this, there's a launch option for it.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 2 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: Done.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**